specs/test/testplan.md

```diff
@@ -168,16 +199,55 @@ It should cover the basic function to pass, to fail, and error conditions.
 ---
 
 ## stress test
+
 ### protocol level stress
+
 * test case # : STP1
 * description : 
-* test procedure :
-* passing criteria
+* test procedure : increase number of txns in block from 1000 to 10,000, stepwise
+* evaluation: change in block latency per shard/change in transactions per sec, per shard.
+* dependency
+* note
+* automated?
+---
+* test case # : STP2
+* description : increasing number of nodes in a shard
+* test procedure : increase number of nodes in a shard from 100 to 5000
+* evaluation: change in latency per shard/change in transactions per sec, per shard.
 * dependency
 * note
 * automated?
+---
+* test case # : STP3
+* description :  epoch change with increasing number of nodes in shard
+* test procedure : initiate epoch change with increasing number of nodes in a shard from 100 to 1000
+* evaluation: latency in leader election
+* dependency
+* note
+* automated?
+---
+
+##epoch change stress
+* test case # : EC1
+* description : hight waiting nodes
+* test procedure : for 5 shards having 200 nodes each initiate epoch change with 10,000 nodes waiting
+* evaluation: latency in leader election/resharding
+* dependency
+* note
+* automated?
+---
 
 ### networking level stress
+
+* test case # : NT1
+* description : 
+* test procedure : start consensus with a peer-p2p network of 50 peers, increase it to 1000 peers, send 1 ping message every 10 secs.
```

Comment:
for stress, we probably want to stress more, like 1 ping message every 100 ms?

Reply:
Done